### ISSUE TITLE
Hey! I am learning to code in school.

### ISSUE THREAD

**Issue by dylshie** [theabbie/Youtube-Uploader-Bot#8]
Hey! I am learning to code in school and i wanted one of my projects to be a bot which would upload videso for me. I was wondering if you wanted to go through how your code works and how to use it?!

Br, Dylan

Contact info 
dyl#2854 on discord

**Comment by theabbie** [comment]
@dylshie this bot can upload to YouTube but videos will be locked automatically, it might work for demonstration in school, but not as an actual bot, if you are stuck somewhere in tutorial, I can help, thank you.

**Comment by dylshie** [comment]
Yes please do! Also is the bot able to input specific youtube urls for example? Like ones i choose myself? If so, would there be a way to upgrade the code to not lock the channel right away?

**Comment by theabbie** [comment]
@dylshie no way way to prevent video from getting locked, you can input your own videos, not youtube links, but download links to any video will work.
